lib/charms/observability_libs/v0/juju_topology.py

```diff
@@ -87,7 +87,11 @@ def __init__(self, uuid: str):
 
 
 class JujuTopology:
-    """JujuTopology is used for storing, generating and formatting juju topology information."""
+    """JujuTopology is used for storing, generating and formatting juju topology information.
+
+    DEPRECATED: This class is deprecated. Use `pip install cosl` and
+    `from cosl.juju_topology import JujuTopology` instead.
```

Comment:
Can we leave this deprecation docstring out until we decide on an actual cut-over date? The thought was sort-of "once PYDEPS is in a stable-ish channel"

Reply:
I'll merge as-is because I need this asap for the load test.
Let's drop it from the lib and let CI take care of the rest?